.jscsrc

```diff
@@ -2,5 +2,8 @@
   "preset": "google",
   "validateLineBreaks": "LF",
   "validateIndentation": 2,
-  "excludeFiles": ["node_modules/**"]
+  "excludeFiles": ["node_modules/**"],
+  "disallowSpacesInAnonymousFunctionExpression": {
+    "beforeOpeningRoundBrace": null
+  }
```

Comment:
Not sure if this works, but if it does this would be better:

```json
"requireSpacesInAnonymousFunctionExpression": {
    "beforeOpeningRoundBrace": true
}
```